deploy/tests/src/test/java/se/nbis/lega/cucumber/steps/Ingestion.java

```diff
@@ -145,7 +74,7 @@ public Ingestion(Context context) {
     private void ingestFile(Context context) {
         try {
             Utils utils = context.getUtils();
-            utils.publishCEGA(String.format("amqp://%s:%s@localhost:5672/%s",
+            utils.publishCEGA(String.format("amqp://%s:%s@localhost:5670/%s",
```

Comment:
can we make this port configurable as well with the `context` :)

Reply:
Yes, I'll do it.

Reply:
Done.